stdlib/public/core/ManagedBuffer.swift

```diff
@@ -35,42 +35,47 @@ internal func _swift_bufferAllocate(
 /// - Note: Subclasses must not have any stored properties; any storage
 ///   needed should be included in `Header`.
 @_fixed_layout
-open class ManagedBuffer<Header, Element> {
+open class ManagedBuffer<Header, Element: ~Copyable> {
   /// The stored `Header` instance.
   ///
   /// During instance creation, in particular during
   /// `ManagedBuffer.create`'s call to initialize, `ManagedBuffer`'s
   /// `header` property is as-yet uninitialized, and therefore
   /// reading the `header` property during `ManagedBuffer.create` is undefined.
+  @_preInverseGenerics
   public final var header: Header
 
   #if $Embedded
   // In embedded mode this initializer has to be public, otherwise derived
   // classes cannot be specialized.
+  @_preInverseGenerics
   public init(_doNotCallMe: ()) {
     _internalInvariantFailure("Only initialize these by calling create")
   }
   #else
   // This is really unfortunate. In Swift 5.0, the method descriptor for this
   // initializer was public and subclasses would "inherit" it, referencing its
   // method descriptor from their class override table.
+  @_preInverseGenerics
   @usableFromInline
   internal init(_doNotCallMe: ()) {
     _internalInvariantFailure("Only initialize these by calling create")
   }
   #endif
 
+  @_preInverseGenerics
   @inlinable
   deinit {}
 }
 
 @available(*, unavailable)
-extension ManagedBuffer: Sendable {}
+extension ManagedBuffer: Sendable where Element: ~Copyable {}
```

Comment:
Given that (a) we can conform to Sendable in only one way, and (b) that one way is unavailable… do we even need to spell out a `where` clause here?

Reply:
We don't! I went for the unconstrained version purely out of an aesthetic desire to not make the unavailable conformance conditional.